pkg/api/api_test.go

```diff
@@ -62,3 +69,144 @@ func TestNewApiWithInvalidSourceIPExtractor(t *testing.T) {
 	require.Error(t, err)
 	require.Nil(t, api)
 }
+
+func TestApiGzip(t *testing.T) {
+	cfg := Config{}
+	serverCfg := getServerConfig(t)
+	srv, err := server.New(serverCfg)
+	require.NoError(t, err)
+	go func() { _ = srv.Run() }()
```

Comment:
`go srv.Run()` here is equivalent

Reply:
But the linter complained :(

Reply:
That's.... weird. Is it just because it didn't like the lack of return value? I feel like `go srv.Run()` is better from a reading perspective for anyone who doesn't know annoying go quirks